src/components/Label/Label.scss

```diff
@@ -126,7 +125,6 @@
     }
 
     70% {
-      transform: rotate(-5deg);
```

Comment:
we sure we want to remove the rotate? we showed it to meirav?

Reply:
+1